changelog/unreleased/kong/fix-acme-username-password-auth.yml

```diff
@@ -0,0 +1,3 @@
+message: "**ACME**: Fixed an issue where username and password were not accepted as valid authentication methods."
+type: bugfix
+scope: Plugin
```

Comment:
we probably need an addtional changelog for the bumped dependency, as I
heard from @AndyZhang0707

Reply:
Done in https://github.com/Kong/kong/pull/13496/commits/61a36dc6c7c9e7069847fc6885d730b7baa88b2c